### PR TITLE
Fixes #3446 - setup OpenTelemetry

### LINKED ISSUE #3446: Setup OpenTelementry with AWS CloudWatch
Preparing for the first stages of @mattwiller's [Observability](https://github.com/medplum/medplum/discussions/2759) plan

1. Setup the OpenTelementry Node.js SDK: https://opentelemetry.io/docs/instrumentation/js/getting-started/nodejs/#instrumentation
2. Setup the AWS CloudWatch EMF Exporter: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsemfexporter

### PR DESCRIPTION
**Pull request by codyebberson**

### CODE REVIEW

File: packages/docs/docs/self-hosting/opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 900
+---
+
+# OpenTelemetry Support
+
+This page describes how to add the Datadog agent to your ECS Fargate tasks. Adding Datadog allows you to collect metrics from all containers.
```
Comment (ThatOneBro) [inline packages/docs/docs/self-hosting/opentelemetry.md:7]: Think this line might be leftover copypasta?

File: packages/docs/docs/self-hosting/opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+npx cdk diff -c config=my-config-file.json
+```
+
+Run `deploy` to apply changes:
+
+```bash
+npx cdk deploy -c config=my-config-file.json
+```
+
+For more details about the Datadog / Amazon ECS Fargate integration, refer to the full documentation: [https://docs.datadoghq.com/integrations/ecs_fargate/](https://docs.datadoghq.com/integrations/ecs_fargate/)
```
Comment (ThatOneBro) [inline packages/docs/docs/self-hosting/opentelemetry.md:109]: Same here

**Review by ThatOneBro — CHANGES_REQUESTED** [review]
A few things regarding docs cleanup, but overall think it's a good start

File: packages/docs/docs/self-hosting/opentelemetry.md (hunk 1 above)
Comment (codyebberson) [reply packages/docs/docs/self-hosting/opentelemetry.md:7]: Eek!  Good catch!

**Review by mattwiller — APPROVED** [review]

**Review by ThatOneBro — APPROVED** [review]

**Pull request merged**